tools/autograd/templates/VariableType.cpp

```diff
@@ -381,6 +381,17 @@ static bool isFloatingPoint(ScalarType s) {
 }
 
 Tensor & VariableType::s_copy_(Tensor & self, const Tensor & src, bool non_blocking) const {
+  jit::Node* node = nullptr;
+  if(torch::jit::tracer::isTracing()) {
+    auto& graph = jit::tracer::getTracingState()->graph;
+    // if you have no views of self, then an in place copy is equivalent to
+    // making sure we expand src to the same size as self
+    node = graph->create(jit::aten::expand_as, /*outputs=*/0);
+    jit::tracer::addInputs(node, "src", src);
+    jit::tracer::addInputs(node, "self", self);
+    graph->appendNode(node);
+    jit::tracer::ensureUnique("left-hand side assignment", self);
```

Comment:
Well this might be very misleading if someone does `x.copy_(y)`, so it might be worth saying that it's a `problem with copy (possibly due to an assignment)`